addons/mail/static/src/models/messaging_notification_handler/messaging_notification_handler.js

```diff
@@ -126,6 +128,13 @@ function factory(dependencies) {
             await this.async(() => Promise.all(proms));
         }
 
+        _handleNotificationAttachmentDeleted(payload) {
+            const attachment = this.messaging.models['mail.attachment'].findFromIdentifyingData({id: payload.id});
```

Comment:
lint: space inside inline object in js
But instead directly use the payload as the identifying data.